packages/apollos-church-api/config.yml

```diff
@@ -75,6 +77,7 @@ ROCK_MAPPINGS:
     UniversalContentItem:
       EntityType: ContentChannelItem
     WeekendContentItem:
+      ContentChannelId: [5]
```

Comment:
is there a reason why this was added? I defined it already below.

Reply:
Bad merge on my part